JS/Lesson34/goksel/recipe-app/src/modules/NewRecipe/NewRecipeForm/index.jsx

```diff
@@ -0,0 +1,151 @@
+                    minLength: {
+                        value: 50,
+                        message: "Please enter Instractions at least 50 characters long"
+                    }
+                })}
+            />
+            {errors.strInstructions && (
+                <span className="error-message">{errors.strInstructions.message}</span>
+            )}
+
+            <label htmlFor="strMealThumb">Upload Your Image</label>
+            <input
+                id="strMealThumb"
+                type="file"
+                {...register("strMealThumb")}
+            />
+
+            <label htmlFor="strTags">Main Product</label>
+            <textarea
+                id="strTags"
+                {...register("strTags", {
+                    required: "Tags are required",
+                    minLength: {
+                        value: 5,
+                        message: "Please enter at least 1 tag"
+                    }
+                })}
+            />
+            {errors.strTags && (
+                <span className="error-message">{errors.strTags.message}</span>
+            )}
+
+            <label htmlFor="strYoutube">Share Youtube Link</label>
+            <input
+                id="strYoutube"
+                type="text"
+                {...register("strYoutube")}
+            />
+
+            <label htmlFor="strIngredientsAndMeasures">Ingredients and Measures</label>
+            <textarea
+                placeholder="Milk - 500ml, eggs - 2, sugar - 2tbsp"
+                id="strIngredientsAndMeasures"
+                {...register("strIngredientsAndMeasures", {
+                    required: "Ingredients and Measures are required",
+                    minLength: {
+                        value: 10,
+                        message: "Please enter at least 5 Ingredients and Measures"
+                    }
+                })}
+            />
+            {errors.strIngredientsAndMeasures && (
+                <span className="error-message">{errors.strIngredientsAndMeasures.message}</span>
+            )}
+
+            <label htmlFor="strSource">Source of Your Meal</label>
+            <input
+                id="strSource"
+                type="text"
+                {...register("strSource")}
+            />
+
+            <label htmlFor="strImageSource">Source of Your Image</label>
+            <input
+                id="strImageSource"
+                type="text"
+                {...register("strImageSource")}
+            />
+
+            <label htmlFor="strCreativeCommonsConfirmed">Is Creative Commons Confirmed</label>
+            <select id="strCreativeCommonsConfirmed" {...register("strCreativeCommonsConfirmed")}>
+                <option value="null">Null</option>
+                <option value="true">Yes</option>
+                <option value="false">No</option>
+            </select>
+
+            <label htmlFor="dateModified">Date</label>
```

Comment:
I'd recommend removing this input field as we usually set this date programmatically once the user submits the input. Instead, add it to the  onSubmit function (use Date.now() function to generate current date)